com/clover/sdk/FdParcelable.html

```diff
@@ -96,7 +96,7 @@ <h2 title="Class FdParcelable" class="title">Class FdParcelable&lt;V&gt;</h2>
 </div>
 <div class="contentContainer">
 <ul class="inheritance">
-<li><a href="http://d.android.com/reference/java/lang/Object.html?is-external=true" title="class or interface in java.lang">java.lang.Object</a></li>
+<li>java.lang.Object</li>
```

Comment:
It would be nice if these links were still functional. 
When it's a third-party dependency, there's usually a classpath issue when the javadoc is generated. You'll see a `cannot find symbol` or similar error in the log when this occurs.

A second issue is that the links in the previous version were HTTP, so the browser blocks the link due to it being a mixed content resource. Can these be changed to HTTPS with a javadoc flag? It would be a much better reader experience.